gym/lib/gym/runner.rb

```diff
@@ -100,6 +100,20 @@ def build_app
       mark_archive_as_built_by_gym(BuildCommandGenerator.archive_path)
       UI.success("Successfully stored the archive. You can find it in the Xcode Organizer.") unless Gym.config[:archive_path].nil?
       UI.verbose("Stored the archive in: " + BuildCommandGenerator.archive_path)
+
+      post_build_app
+    end
+
+    # Post-processing of build_app
+    def post_build_app
+      command = BuildCommandGenerator.post_build
+      print_command(command, "Generated Post-Build Command") if FastlaneCore::Globals.verbose?
```

Comment:
@allewun I _did_ just catch one bug there 😊 Things go 💥 if the `command` array here is empty since... Putting an `unless command.empty?` around this fixed it for me... Do you have this same issue?

Reply:
Nice catch! I added a guard here.